tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -92,6 +92,71 @@ class DatasetV2(tracking_base.Trackable, composite_tensor.CompositeTensor):
   A `Dataset` can be used to represent an input pipeline as a
   collection of elements (nested structures of tensors) and a "logical
   plan" of transformations that act on those elements.
+  
+  
+For example:
+
+    ```python
+    tf.print("tf.keras.model.fit() can take a single tf.data.Dataset for training features and labels. \
```

Comment:
Should also mention that if you use tf.keras.layers.Input(..., name=name) the dataset can also return a dictionary with the names of the inputs as keys.

Reply:
I was unaware of that capability!  But I’m excited to try it out.  I appreciate the response and I’ll edit and resubmit.

Reply:
I was unaware of that capability!  But I’m excited to try it out.  I appreciate the response and I’ll edit and resubmit.

Reply:
Please ping here when you want me to take another look :-)